src/queries/lpFills.ts

```diff
@@ -117,4 +117,8 @@ export default async function getLpFills({
       alias: lp.alias || undefined,
     };
   });
+
+  lpFills.sort((a, b) => b.filledAmountValueUsd.comparedTo(a.filledAmountValueUsd));
```

Comment:
now that we sort here, i think we can remove the sorting on line 74

Reply:
we will do this in a follow up to get the fix out - thanks again!

Reply:
@niklasnatter fixed in https://github.com/chainflip-io/chainflip-info-bot/pull/125